src/components/pickers/customPickerStyles.ts

```diff
@@ -0,0 +1,34 @@
+const customPickerStyles = {
+  control: (provided: any) => ({
+    ...provided,
+    border: 'none',
+    boxShadow: 'none',
+    minHeight: '100%',
+    height: '100%',
+    fontSize: '14px',
+    overflow: 'hidden',
+    outline: 'none',
+    padding: '0px',
+  }),
+  menuList: (base: any) => ({
+    ...base,
+    // kill the white space on first and last option
```

Comment:
@KarolinaKopacz is this comment necessary?

Reply:
I think yes, for developer who don't know exactly how styling works in react-select it say enough and them no one will want to remove it because padding = 0px

Reply:
ok